comments/forms.py

```diff
@@ -29,4 +30,7 @@ class Meta:
 
     def clean_content(self) -> str:
         content = self.cleaned_data.get("content")
-        return html_sanitizer(content)
+        if not_empty_comment(content):
```

Comment:
I think it would be clearer
```suggestion
        if is_empty_comment(content):
            raise forms.ValidationError(_("Empty comment"))
            
        return html_sanitizer(content)
```